### PR TITLE
docs(relay): Update directory creation when enabling spooling

### PR DESCRIPTION
**Pull request by iker-barriocanal**
See https://github.com/getsentry/relay/pull/2636.

### CODE REVIEW

File: src/docs/product/relay/options.mdx [hunk 1]
```diff
@@ -62,13 +62,13 @@ These settings fine tune caching of project states.
...
-| spool.envelopes.min_connections | Integer | default: `10`    | The minimum number of connections in the pool to manage on-disk spooling.                                                    |
+| key                             | type    | value            | description                                                                                                                                                                                                             |
+| ------------------------------- | ------- | ---------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| spool.envelopes.path            | String  | default: `null`  | The path to the spool file used to save envelopes to disk. If provided, Relay will create the directory structure and enable on-disk spooling. Note that `~` is interpreted as a literal and not as the home directory. |
```
Comment (olksdr) [inline src/docs/product/relay/options.mdx:67]:
```suggestion
| spool.envelopes.path            | String  | default: `null`  | The path to the spool file used to save envelopes to disk. If provided, enables on-disk spooling. Relay will create the directory structure and enable on-disk spooling. Note that `~` is interpreted as a literal and not as the home directory. |
```

**Review by olksdr — CHANGES_REQUESTED** [review]
I still think we should keep the sentence as before and just add the part about the folders autocreate. See the suggested changes.

File: src/docs/product/relay/options.mdx (hunk 1 above)
Comment (iker-barriocanal) [reply src/docs/product/relay/options.mdx]: Reworded to something similar, let me know what you think. I don't like duplicating the "enable on-disk" part.

**Review by vivianyentran — APPROVED** [review]
lgtm

**Pull request merged**